### ISSUE TITLE
Feature: Seitenwechsel über variable

### ISSUE THREAD

**Issue by BenAhrdt** [ioBroker/ioBroker.vis-2-widgets-jaeger-design#54]
Ist es möglich einen Seitenwechsel / Seitenaufruf über ein Skript zu machen?

Hintergrund:
Öffnen einer Tür => Sprung zur Seite der Tür.

Müll muss raus => Sprung zur Seite der Mülltonnen
Usw.

**Comment by BenAhrdt** [comment]
Habe ganz vergessen, dass das über die Variablen der vis2 gehen sollte.
Allerdings geht es nicht.
Warum auch immer.
Die Daenpunkte bleoben unbestätigt.
![image](https://github.com/user-attachments/assets/e255b6a6-f9ae-46e0-b006-854e4c22b0c0)

**Comment by obodeldog** [comment]
Yes, I know, it's not working so well yet. But try it as shown in the picture.

![Image](https://github.com/user-attachments/assets/fc40e8a6-2f8d-4811-99a8-6ef09a530031)

**Comment by obodeldog** [comment]
in englisch:

Is it possible to perform a page switch or page call via a script?

Background: Opening a door => Jump to the door page.

Trash needs to be taken out => Jump to the trash bins page. And so on.

It should actually work via the variables of vis2. However, it doesn't work. For whatever reason, the data points remain unconfirmed.

**Comment by BenAhrdt** [comment]
OK. I actually use the fully browser and change the page with the whole url

**Comment by GermanBluefox** [comment]
So you can implement it:
- Select the desired page in vis-2 (runtime)
- Get the value of state vis-2.0.control.data into following JavaScript script:
```
setState('vis-2.0.control.data', 'Here is the value of vis-2.0.control.data when you clicked it');
setState('vis-2.0.control.instance', '*'); // you can place here your specific browser instance
setState('vis-2.0.control.command', 'changeView');
```

When this code is executed, the vis-2 jumps to the desired view.
